### PR TITLE
Check dangerpoint get permission

### LINKED ISSUE #6: Check permissions if a DangerPoint is fetched by the server
get($id) and get_all()

ROLE_ADFC_ADMIN read everything
ROLE_SCHOOL_ADMIN and/or ROLE_SCHOOL_REVIEWER shall read every point of a school.
Other: see only own DangerPoints

### PR DESCRIPTION
**Pull request by tabacha**
fixes #6

### CODE REVIEW

File: src/AppBundle/Controller/DangerPointController.php [hunk 1]
```diff
@@ -72,6 +72,12 @@ public function getAllAction()
     {
         if ($this->get('security.authorization_checker')->isGranted('ROLE_ADMIN')) {
             $restresult = $this->getDoctrine()->getRepository('AppBundle:DangerPoint')->findAll();
+        } elseif (
```
Comment (jetwell) [inline src/AppBundle/Controller/DangerPointController.php]: The swagger documentation for all functions in this file must include the 404 responses. They aren't all documented.

File: src/AppBundle/Controller/DangerPointController.php [hunk 2]
```diff
@@ -72,6 +72,12 @@ public function getAllAction()
...
+            $this->get('security.authorization_checker')->isGranted('ROLE_SCHOOL_ADMIN') or
```
Comment (jetwell) [inline src/AppBundle/Controller/DangerPointController.php]: This check and the checks below could be extracted to a private function to make the code more concise and readable. Something like: `$this->userHasOneOfRoles('ROLE_SCHOOL_ADMIN', 'ROLE_SCHOOL_REVIEW', 'ROLE_STUDENT')`

File: src/AppBundle/Controller/DangerPointController.php (hunk 1 above)
Comment (tabacha) [reply src/AppBundle/Controller/DangerPointController.php]: this was not part of the changed code, but I fixed it

File: src/AppBundle/Controller/DangerPointController.php (hunk 2 above)
Comment (tabacha) [reply src/AppBundle/Controller/DangerPointController.php]: fixed

**Pull request merged**